docs/rules/jsx-max-props-per-line.md

```diff
@@ -58,4 +58,4 @@ The following patterns are not considered warnings:
 
 ## When not to use
 
-If you are not using JSX then you can disable this rule.
+If you are not using JSX or you already use the [`max-len` rule](http://eslint.org/docs/rules/max-len) then you can disable this rule.
```

Comment:
Generally this section of rules tells you when it's useless to use the rule. Using `max-len` doesn't make it useless.